sympy/core/tests/test_arit.py

```diff
@@ -2307,3 +2307,8 @@ def test__neg__():
 
 def test_issue_18507():
     assert Mul(zoo, zoo, 0) is nan
+
+
+def test_issue_17130():
+    e = Add(b, -b, I, -I, evaluate=False)
+    assert e._eval_is_zero() is None # ideally this would be True
```

Comment:
The test should be for `e.is_zero` rather than `e._eval_is_zero`

Reply:
ok, I have fixed it.